unicodetools/data/security/dev/confusables.txt

```diff
@@ -577,10 +577,10 @@ FF07 ;	0027 ;	MA	#* ( ＇ → ' ) FULLWIDTH APOSTROPHE → APOSTROPHE	# →’
 2018 ;	0027 ;	MA	#* ( ‘ → ' ) LEFT SINGLE QUOTATION MARK → APOSTROPHE	# 
 2019 ;	0027 ;	MA	#* ( ’ → ' ) RIGHT SINGLE QUOTATION MARK → APOSTROPHE	# 
 201B ;	0027 ;	MA	#* ( ‛ → ' ) SINGLE HIGH-REVERSED-9 QUOTATION MARK → APOSTROPHE	# →′→
+05F3 ;	0027 ;	MA	#* ( ‎׳‎ → ' ) HEBREW PUNCTUATION GERESH → APOSTROPHE	# 
```

Comment:
FYI it seems like the order of entries is not stable...

Reply:
Right, that's been something I've been meaning to get to.